source/configuration_manual/mail_location/imapc/index.rst

```diff
@@ -100,24 +111,35 @@ Connection Settings
 
    **Optimizations**
 
-   ``fetch-bodystructure``
+   ``no-fetch-bodystructure``
+
+     Disable fetching of ``BODY`` and ``BODYSTRUCTURE`` which is available by
+     default.
```

Comment:
Maybe clearer:

Regenerated the IMAP ``BODY`` and ``BODYSTRUCTURE`` replies from the message body instead of fetching them from the remote server.